Makefile

```diff
@@ -40,6 +40,25 @@ $(release_drivers): sgp-common/sgp_git_version.c
 	cd release && zip -r "$${pkgname}.zip" "$${pkgname}" && cd - && \
 	ln -sf $${pkgname} $@
 
+release/sgp40:
```

Comment:
```suggestion
release/sgp40: sgp-common/sgp_git_version.c
```

Reply:
.. I think it also misses a dependecy on `docs/Application_Note_SGP40.pdf` but I don't see the make target off a quick glance

Reply:
I added a docs target and added the dependencies accordingly.